src/scripts/main.js

```diff
@@ -1,7 +1,92 @@
-'use strict';
+const Game = require('../modules/Game.class');
+const game = new Game();
 
-// Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+const container = document.querySelector('.container');
+const gameField = document.querySelector('.game-field');
+const button = container.querySelector('.button');
+const gameScore = container.querySelector('.game-score');
 
-// Write your code here
+const fieldCells = [...gameField.querySelectorAll('.field-row')].map((row) => [
+  ...row.children,
+]);
+
+const messages = {
+  idle: container.querySelector('.message-start'),
+  lose: container.querySelector('.message-lose'),
+  win: container.querySelector('.message-win'),
+};
+
+function fillGameField(state) {
+  state.forEach((row, i) => {
+    row.forEach((cell, j) => {
+      const cellElement = fieldCells[i][j];
+
+      cellElement.className = cell
+        ? `field-cell field-cell--${cell}`
+        : 'field-cell';
+      cellElement.textContent = cell || '';
+    });
+  });
+}
+
+function showMessage() {
+  const gameStatus = game.getStatus();
+
+  Object.keys(messages).forEach((key) => {
+    const messageElement = messages[key];
+
+    if (messageElement) {
+      messageElement.classList.toggle('hidden', key !== gameStatus);
+    }
+  });
```

Comment:
The `showMessage` function toggles the visibility of messages based on the game status. It's a good practice to separate concerns by having a function that only determines which message to show, and another one to handle the display logic.